package.json

```diff
@@ -10,7 +10,7 @@
   },
   "main": "index.js",
   "bin": {
-    "calypso-gutenberg-sdk": "./bin/sdk-cli.js"
+    "build-gutenberg": "./bin/sdk-cli.js"
```

Comment:
This will end up looking funny when used with `npx` if it's equal to yargs-command:

```
npx <bin> <yargs-command>
```

would now become:
```
npx build-gutenberg build-gutenberg --editor-script=...
```

We can refactor `build-gutenberg` yargs command out and then we'd have:

```
npx build-gutenberg --editor-script=...
```

That assumes we're not going to have other methods than `build` (we might end up having `create` boilerplate?)

This might also feel little too generic when typed inside Jetpack folder. They might have their own script called `build-gutenberg` already (e.g. `yarn build-gutenberg`). Some unique `bin` name would be nice — maybe `calypso`?